lib/util/http/github.spec.ts

```diff
@@ -142,6 +142,18 @@ describe('util/http/github', () => {
       const trace = httpMock.getTrace();
       expect(trace).toHaveLength(1);
     });
+    it('should handle Github App installation endpoint response format', async () => {
+      const url = '/some-url';
+      httpMock
+        .scope(githubApiHost)
+        .get(url)
+        .reply(200, {
+          repositories: ['a'],
+        });
+      const res = await githubApi.getJson('some-url');
+      expect(res).toBeDefined();
+      expect(res.body).toEqual(['a']);
+    });
```

Comment:
Test should reside here: https://github.com/renovatebot/renovate/blob/7c4a71b679453af716fdb4fb96425dca1e712188/lib/platform/github/index.spec.ts#L161